pkg/plugins/manager/pipeline/bootstrap/doc.go

```diff
@@ -0,0 +1,16 @@
+// Package bootstrap defines the second stage of the plugin loader pipeline.
+//
+// The Bootstrap stage must implement the Bootstrapper interface.
+// - Bootstrap(ctx context.Context, src plugins.PluginSource, bundles []*plugins.FoundBundle) ([]*plugins.Plugin, error)
+//
+// The Bootstrap stage is made up of the following steps (in order):
+// - Construct: Create the initial plugin structs based on the plugin(s) found in the Discovery stage.
+// - Decorate: Decorate the plugins with additional metadata.
```

Comment:
Yeah great observation. I struggled with this a bit and I think it makes sense.

I had initially thought that the Bootstrap struct is probably what most people will just use considering the funcs are pretty flexible - however, to do this properly since we have the top-level interface, it makes sense to move this stuff around 👍